src/main/java/com/myworkspace/reportapp/entity/report/Report.java

```diff
@@ -0,0 +1,96 @@
+package com.myworkspace.reportapp.entity.report;
+
+import com.myworkspace.reportapp.entity.customer.Customer;
+import com.myworkspace.reportapp.entity.customer.Employee;
+import com.myworkspace.reportapp.entity.device.Device;
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+@Table(name = "reports")
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+public class Report {
+
+    @Id
+    private UUID id;
+    private String title;
+    private LocalDate reportDate;
+    private LocalDateTime startWork;
+    private LocalDateTime endWork;
+    private int overallWorkingHours;
+
+    @Column(length = 4000)
```

Comment:
You've set the length attribute of the description column to 4000 characters. Be mindful of the maximum column length allowed by your database system. If 4000 characters is appropriate for your use case, then it's fine. If not, adjust the length accordingly.

Reply:
I think 4000 characters is enough to describe your work in a report. The limit value for varchar is 65535 which is too big and the default value is 255 which is too small for description.